server/schema/published_schedule.sql

```diff
@@ -0,0 +1,16 @@
+DROP TABLE [IF EXISTS] published_schedule;
```

Comment:
The brackets around [IF EXISTS] are mainly for readability in tutorials, but they're not actually part of the syntax. Try removing the brackets to remove the eslint issues!